drivers/entropy/Kconfig.nrf54l_cracen

```
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config ENTROPY_NRF54LCRACEN_CTRDRBG
+	bool "nRF54L entropy driver based on the CRACEN CTR DRBG driver"
```

```suggestion
	bool "nRF54L entropy driver based on the CRACEN CTR_DRBG driver"
```
CTR_DRBG is the correct naming convention according to NIST SP800-90A

---

I guess all `CTRDRBG`  and `ctrdrbg` be `CTR_DRBG` and `ctr_drbg`, too?

---

AFAIK we cannot have "_" in DTS names. Anyhow, the name is long enough, so I hope it is ok without it there :)